zos_concepts/data_transfer/archive_copy_unarchive_restore/README.md

```diff
@@ -0,0 +1,75 @@
+# Transfer Data Sets Using Ansible
+
+This project provides sample playbooks and roles which can be used to transfer
+data sets between z/OS systems using Red Hat Ansible Certified Content for IBM Z.
+
+following core modules are used to accomplish these set of tasks:
```

Comment:
Change these for this

Reply:
Grammatically, it would be "this set of tasks" since the 'this' refers to a singular 'set' of tasks rather than the multiple 'tasks'